pkg/resources/resources.go

```diff
@@ -306,13 +306,13 @@ func AwaitResourceDeletionOrFail(ctx context.Context, t *testing.T, cfg *envconf
 // and then provides basic CRD tests for the resource.
 type ResourceTestConfig struct {
 	Kind              string
-	Obj               k8s.Object
+	Obj               *k8s.Object
```

Comment:
I guess if this were used somewhere already it would be a breaking change. But since we are early in development, this ok to change.

Btw, do we have versioning enabled for this repo?

Reply:
Yep. we can do semver based releases.